app/batid/admin.py

```diff
@@ -70,6 +71,7 @@ def get_urls():
         my_urls = [
             path(r"worker/", admin.site.admin_view(worker)),
             path(r"export_ads/", export_ads),
+            path(r"export_contributions/", export_contributions),
```

Comment:
Pour préparer d'autres API, je propose de changer ces 2 paths par:
```suggestion
            path(r"ads/export/", export_ads),
            path(r"contribution/export", export_contributions),
```

Reply:
Le problème est que les adresses de type ads/[quelquechose]/ et contributions/[quelquechose]/ on tombe sur une 404 car django admin tente de trouver l'ADS avec ID quelque chose.